### DISCUSSION TITLE
Ability to change Port Number

### DISCUSSION THREAD

**Discussion by chanx** [josephdadams/TallyArbiter#616] [Ideas]
Hello, I think we need to change the default port number or at least add documentation about that in readme.

I faced problems TallyArbiter is crash because port 4455 is being used by OBS Studio Default WebSocket Server. and found that problem by seeing the log.

OBS Studio is use by many streamers around the world, so maybe we need change default port number to other port number to avoid that conflict or we just add the warning message in readme or doc only?

**Comment by josephdadams** [comment]
We were using it first. They should change. 😂

**Reply by chanx** [reply]
I don't think that's possible, because their user base is far more than we here. I think we need to change that? Do you want me to do it?
Or if we don't change it, maybe we can include some notes on Readme file?

**Comment by JTF4** [comment]
If you want to change the port number, it can be changed in the config file. 

I don't think changing the default port is a necessary change. Doing so will break backwards compatibility with all listener clients. 

If a port issue does come up, it's very easy to change the websocket port back to something like 4444 in settings. 

I'm all for adding some documentation for it, but I don't think changing the default port is the right way to go.
